integrationtest/cdi/pom.xml

```diff
@@ -0,0 +1,244 @@
+<!--
+
+     Copyright 2017 - 2023 The ModiTect authors
```

Comment:
Copyright notice of all new files should be 2024

Reply:
Ok with that, but since the license is checked on every file with goal `license:format`, I'll have to change the source file in ~/etc folder and update all copyright mention in the project. Should probably be done in another PR, don't you think?

Reply:
Yes, you're absolutely right

Reply:
I can make the PR with cleanup and update if you want.

Reply:
That would be amazing, thank you.

Reply:
When changing this, can we get rid of the year altogether and move to a header like [here](https://github.com/kcctl/kcctl/pull/429)? Will make things simpler down the road.

Reply:
created PR #242